src/Model/Subscription.php

```diff
@@ -136,25 +133,25 @@ public static function getPrivateVapidKey(): string
 	/**
 	 * Prepare push notification
 	 *
-	 * @param int $nid
+	 * @param Notification $Notification Notification instance
```

Comment:
Once more.
```suggestion
	 * @param Notification $Notification
```

Reply:
I told you in another PR, I like to capitalize structured variables in a sea of scalar variables to make them stand out. You can make them all even, I won't stop you, but I feel like we're losing a visual marker of their specificity.

Reply:
Hmm, okay. I personally would name them `$imageInstance` to mark them out. Then I can still have scalar variables like `$uid`. So I better rename them back to capitalized versions to go conform with your coding-convention?

Reply:
It's my own personal convention, I don't want to impose it on anyone else (unlike PSR-2). I used it in my own work but I'm not aware about anyone else using it so it should be pretty disparate across the code base.